src/test/CharterManager-unit.t.sol

```diff
@@ -154,14 +156,17 @@ contract CharterManagerTest is TestBase {
         vat.init(ilk);
         vat.file("Line", 100 * CEILING * 1e27);
         vat.file(ilk, "line", CEILING * 1e27);
-        vat.file(ilk, "spot", 1e36);
+        vat.file(ilk, "spot", 1e27);
+
+        spotter = new Spotter(address(vat));
+        spotter.file(ilk, "mat", 1.5 * 1e27);
```

Comment:
Nit on number formatting throughout this file--usually we'd write `1.5 * RAY`, etc instead of everywhere using a mathemetical literal everywhere for fixed point precision.

Reply:
Yeah the test file is based on CropJoin. Let's consider to refactor after this pr is approved, so we don't make it harder for folks to review.